kafka_consumer/README.md

```diff
@@ -43,20 +43,20 @@ The Kafka-consumer check does not include any service checks.
 
 ## Further Reading
 
-* [Monitoring Kafka performance metrics][120]
-* [Collecting Kafka performance metrics][121]
-* [Monitoring Kafka with Datadog][122]
+* [Monitoring Kafka performance metrics][1110]
+* [Collecting Kafka performance metrics][1111]
+* [Monitoring Kafka with Datadog][1112]
 
 
 [111]: https://raw.githubusercontent.com/DataDog/integrations-core/master/kafka_consumer/images/kafka_dashboard.png
```

Comment:
Can you fix those numbers?

Reply:
those numbers are correct because of the "duplicated" integrations issue